### DISCUSSION TITLE
Bruhwalker access

### DISCUSSION THREAD

**Discussion by SultanTheGoat** [community/community#38442] [Other Feature Feedback, Questions, & Ideas]
Dear Bruhwalker administration,

I have been desperately trying to find a way of contact with any member of the bruhwalker team. I have not heard anything but good recommendations from the community and have explored many scripts yet couldn’t find anyone as reliable as bruhwalker’s reputation. I wanted to get in contact with you and request a full time membership. I would like to be a never lasting customer and I hope that you can accept me. Please contact me on me email [removed by moderator]
I tried several time and I have been trying to get a code in order to register with bruhwalker for a long period of time and I hope you get in touch with me.

Sincerely, Sultan Alkhoori

**Comment by david-engelmann** [comment]
I'm not sure this is the right location for this inquiry. If they have a repo, it's best to post this there.
